runtime/doc/treesitter.txt

```diff
@@ -1321,7 +1321,9 @@ LanguageTree:is_valid({exclude_children})            *LanguageTree:is_valid()*
 
     Parameters: ~
       • {exclude_children}  (`boolean?`) whether to ignore the validity of
-                            children (default `false`)
+                            children (default `false`). `is_valid(false)` can
+                            be slow because it runs injection on the full
+                            source.
```

Comment:
And this? I think that belongs to the top-level docs, not the parameter?

Reply:
fixed